templates/nginx.conf

```diff
@@ -77,7 +77,7 @@ server {
     }
 
     # backend
-    location ~ ^/(api|pictrs|feeds|nodeinfo|.well-known) {
+    location ~ ^/(api|pictrs|feeds|inbox|nodeinfo|.well-known) {
```

Comment:
@Nutomic Is this one necessary? Wouldn't federation be not working anywhere if this was required?

Reply:
I'm not really sure if this change helps or hurts, but I can at least confirm that federation is not working the way I'd expect it to on the instance I set up recently using this repo. The main issue is that comment syncing, in that most comments that are visible on other instances are not visible on my own instance when I visit a post.

There's more info here about comment sync problems that I and others have been experiencing: https://lemmy.ml/post/1148327?scrollToComments=true

Reply:
Here's an example of traffic logged by nginx when I visit `mylemmydomain.com/inbox`:

```
[my ip address] - - [05/Jun/2023:15:51:14 +0000] "GET /inbox HTTP/2.0" 404 0 "-" "Mozilla/5.0 [my user-agent]..."
[my ip address] - - [05/Jun/2023:15:51:14 +0000] "GET /inbox HTTP/2.0" 404 0 "-" "Mozilla/5.0 [my user-agent]..."
```

Here's an example of traffic that occurs regularly, with a variety of IPs and user-agent strings, most of which include either "lemmy.ml" or "mastodon.social", that made me think this traffic was probably not meant for the frontend:

```
[a datacenter ip address] - - [05/Jun/2023:15:50:42 +0000] "POST /inbox HTTP/1.1" 400 0 "-" "http.rb/5.1.1 (Mastodon/4.1.2+nightly-20230523; +https://mastodon.social/)"
[a datacenter ip address] - - [05/Jun/2023:15:50:42 +0000] "POST /inbox HTTP/1.1" 400 0 "-" "http.rb/5.1.1 (Mastodon/4.1.2+nightly-20230523; +https://mastodon.social/)"
```

Reply:
Sorry, one more comment: I can see now that the comment syncing behavior is probably related to this. #2825

But I am still wondering why I'm seeing so much traffic on the `/inbox` route for an instance where I am currently the only registered user.

Reply:
Yep this is already handled by `$request_method = POST` above. The inbox is where incoming federation actions from other servers are received (as POST requests).